flink-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaTestEnvironment.java

```diff
@@ -80,6 +82,12 @@ public void createTestTopic(String topic, int numberOfPartitions, int replicatio
 
 	public abstract <T> FlinkKafkaConsumerBase<T> getConsumer(List<String> topics, KeyedDeserializationSchema<T> readSchema, Properties props);
 
+	public abstract <K, V> Collection<ConsumerRecord<K, V>> getAllRecordsFromTopic(
+		Properties properties,
+		String topic,
+		int partition,
+		long timeout);
```

Comment:
nit: the indentation pattern is inconsistent with the other abstract method declarations here.